From ba2fe6533e6b25d9b8479cc56188b82933f0f968 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?=
 <nuclearsandwich@users.noreply.github.com>
Date: Mon, 25 Jan 2021 15:57:48 -0500
Subject: [PATCH 3/3] Update default RMW dependency. (#10)

For dependency alternatives where none is currently available the first
in line is chosen.
---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index b82e69a..4d90367 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,7 +8,7 @@ Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-rolling-rmw-fastrtps-cpp | ros-rolling-rmw-connext-cpp | ros-rolling-rmw-cyclonedds-cpp
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-rolling-rmw-cyclonedds-cpp | ros-rolling-rmw-connext-cpp | ros-rolling-rmw-fastrtps-cpp
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
-- 
2.29.2

